### PR TITLE
Rules list remove item 2317227

### PR DESCRIPTION
**Pull request by MegaChriz**
The following changes were made:
- Plugin class was renamed from RemoveListItem to DataListItemRemove, according to notes listed on https://github.com/fago/rules/pull/102/files;
- A test was added to tests/Integration/Action.

### CODE REVIEW

File: tests/src/Integration/Action/DataListItemRemoveTest.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    $this->assertEquals('Remove list item', $this->action->summary());
+  }
+
+  /**
+   * Tests the action execution.
+   *
+   * @covers ::execute()
+   */
+  public function testActionExecution() {
+    $list = array('One', 'Two', 'Three');
```
Comment (klausi) [inline tests/src/Integration/Action/DataListItemRemoveTest.php]: This should use the PHP short array syntax.

File: tests/src/Integration/Action/DataListItemRemoveTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    $this->action
+      ->setContextValue('list', $list)
+      ->setContextValue('item', 'Two');
+
+    $this->action->execute();
+
+    // The second item should be removed from the list.
+    $this->assertArrayEquals(array('One', 'Three'), array_values($this->action->getContextValue('list')));
```
Comment (klausi) [inline tests/src/Integration/Action/DataListItemRemoveTest.php]: Same here, we just use [] instead of array() in Rules.

**Comment by MegaChriz** [comment]
I converted the code to the PHP short array syntax. Tests are passing locally. I'm not familiar with Travis CI to understand why the build is failing.

**Comment by klausi** [comment]
The test fails where unrelated to your patch, I removed the Xdebug PHP extension during testing. Please merge the 8.x-3.x branch and push.

**Comment by klausi** [comment]
Just tested this pull request locally and it works. Merged, thanks!

**Pull request closed without merging**